Jason/update sso screenshots (#399)
returntocorp/semgrep-docs: docs/semgrep-app/sso.md

> OAuth2/OIDC can be configured via the Semgrep App Dashboard. Please visit the <b>Configure SSO</b> tab under Settings to edit the SSO configuration.

Reviewer: ```suggestion
OAuth2/OIDC is configured through **Semgrep App Dashboard**. 

To edit SSO configuration, click on **Settings > Configure SSO tab**.
```
